lib/utils/check-version.js

```diff
@@ -0,0 +1,41 @@
+const request = require("request-promise");
+const Logger = require.main.require("lib/logger/logger.js");
+const Errors = require.main.require("lib/response/error.js");
+
+function CheckVersion(pkg) {
+    if (typeof pkg === "object") {
+        const pkg_split = pkg.version.split(".");
+        const version = pkg_split[0] + "." + pkg_split[1];
+        const url = "http://registry.npmjs.org/sealious/" + version;
+        let status = "warning", message;
```

Comment:
I don't get that syntax.  Isn't `message` here undefined? What's the type of the `status` variable?